### ISSUE TITLE
Panic! demo timing issues

### ISSUE THREAD

**Issue by joncampbell123** [joncampbell123/dosbox-x#4]
There seem to be minor timing issues with Future Crew Panic! demo that lags the effects behind the music a bit, then the "shadebob" section runs too fast and jumps to the next part with runs too soon (rotating dot objects). That part finishes too early, and the dots just sit there for a good 5 seconds waiting for the next musical cue to become the FC logo.

I tested around a bit with VFRCRATE to see if it was a refresh rate issue. The issue happens no matter what the refresh rate.

**Comment by joncampbell123** [comment]
OK so it's probably not a DOSBox issue at all. This video capture by someone with a 486DX and a scan converter shows the same problems on real hardware:

https://www.youtube.com/watch?v=MYdRj-lHyyo

Notice the same timing problem where the dot object is left waiting for 3-5 seconds in the shape of the logo before finally becoming the "Future Crew" logo.

**Comment by joncampbell123** [comment]
I'm wondering if the problem with Panic! and some more subtle glitches in other demos could be resolved if DOSBox were to support an I/O delay to VGA memory in the same way that it does with general I/O access now. Meaning, if the user enables it in dosbox.conf, DOSBox could slow down VGA memory access to speeds equivalent to the ISA bus on a 386 in a way the older demos expect.

**Comment by joncampbell123** [comment]
Ok. So the latest DOSBox-X is able to run the demo properly all the way through. The graphical glitches during the "scrollers suck" section seem to be gone. The one remaining issue is the shadebob part that runs way too fast. According to other people's captures of the demo on real hardware (486 and Pentium systems), that problem seems to be with the demo itself and it has nothing to do with DOSBox.

I have verified that if I set the vmemdelay to a large value like 550 (extra slow VGA) the shadebobs slow down to a level that allows the rotating dots section after it to time themselves better to the music and fall down in time to become the FC logo.
